Automattic/wp-calypso
# Site Transfers: Add minor improvements to site transfer screen

Related to https://github.com/Automattic/dotcom-forge/issues/2809

## Proposed Changes

This PR:

* Removes the list items for "Upgrades" since it is always only one item on the Transfer screen
* Change the second sentence in the "Content and ownership" section to: “You’ll keep your admin access unless {oldOwnerEmail} removes you.”
* Changes the third sentence to `Your posts on <strong>%(siteSlug)s</strong> will remain authored your account.`
* Change the texts to display the `username` in all cases except the first occurrence that shows `username (email)`.

**Before**:

<img width="805" alt="Screenshot 2023-06-22 at 3 50 26 PM" src="https://github.com/Automattic/wp-calypso/assets/25575134/c468c328-ca71-441d-9804-37c14793d1b8">

**After**:

<img width="824" alt="Screenshot 2023-06-22 at 3 50 52 PM" src="https://github.com/Automattic/wp-calypso/assets/25575134/b3e8cff9-29ca-4576-bb84-e506bda58405">

## Testing Instructions

* Navigate to **Settings > General > Transfer your site** with Calypso Live link
* Select an administrator to transfer your site to (if you don't have an admin on your site, you would need to add it first)
* Proceed to the transfer consent screen and confirm it looks as indicated by design

## Pre-merge Checklist
